DataStructures/Trees/tree.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace DataStructures.Trees
+{
+    class tree
```

Comment:
This will be your "BinaryTree" where you implement PreOrder, PostOrder, and InOrder, and then the BinarySearchTree will inherit from this class.